src/sentry/static/sentry/app/components/events/eventEntries.tsx

```diff
@@ -378,7 +378,7 @@ class EventEntries extends React.Component<Props, State> {
             hasQueryFeature={hasQueryFeature}
           />
         )}
-        {this.renderEntries()}
+        {this.renderEntries(event)}
```

Comment:
why do that? renderEntries has `this`.

Reply:
it's because I did not want to check 2x if event is defined. 

The UI checked if the event is defined in this line:
https://github.com/getsentry/sentry/blob/97c8b0d9f45faa6524adc5fd29460edbdeb32819/src/sentry/static/sentry/app/components/events/eventEntries.tsx#L330 

so it reaches `renderEntries` the event should be already defined.

if I use `this` inside of `renderEntries` I will have to use `event?.` when using some property of event or check again if the event is defined